### ISSUE TITLE
Create ARP header component

### ISSUE THREAD

**Issue by candilemoine** [department-of-veterans-affairs/va.gov-team#97602]
#### User Story or Problem Statement

As a representative, I want to be able to log in and navigate throughout the portal so I can use the tools for my daily workflow.

#### Mockups/Designs

- [header](https://www.figma.com/design/LVCQBuW7a6nfVFNyhA4kv4/ARF---Form-21-22-Design-Explorations?node-id=1894-144400&node-type=section&t=pgSxGYGo5bUHudF5-0)

### Acceptance Criteria/Conditions of Satisfaction

- [ ] a rep will be directed to the VA ARP homepage when they click on the main logo

_Logged out state_
- [ ] a rep will only see `Get help` and `Sign in`
- [ ] a rep will not see the main navigation

_Logged In state_
- [ ] a rep will see the navigation 
- [ ] a rep will see their profile name
- [ ] a rep will be able to use the drop down to go to their Profile or Logout

#### Accessibility (A11y) Testing 
- Refer to existing [VA design system](https://design.va.gov/components/form/)
- [ ] Test with either [WAVE](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US&utm_source=ext_sidebar) or [AXE](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US&utm_source=ext_sidebar) extension
- As you manually test, look for the following:
   - [ ] color contrast errors
   - [ ] keyboard navigation
     - When navigating with a keyboard, content should flow from left to right
   - [ ] heading order 
     - Headings should be in order, so h1, then h2, h3. This is so the screen reader can let the user know the natural order of the elements in page. The WAVE tool has a headings tab that will display the order.
   - [ ] proper tag elements
     - Since we are working with forms, make sure components have the proper tags, i.e.:
 <label for="lastName"...>Last Name</label> <input id="lastName"... />
- [ ] test with a screen reader 
  - Voiceover is built in with mac, use NVDA for windows

**Comment by seipeljl** [comment]
@candilemoine Per discussion with @eheintzelman we are thinking we will not show "Get Help" for the unauthenticated view.  Will update Figma designs to reflect this.

**Comment by jquispe-oddball** [comment]
@candilemoine Just FYI, ### we do not have a page for the "Get Help" link for the authenticated view, so will just add in an empty link for now. Whenever the Get Help page is built, I will have to go back into the header and update the link to include the url.
cc: @seipeljl

**Comment by candilemoine** [comment]
@jquispe-oddball @nihil2501 Should we breakout the PRs to be distinct: 
- header + navigation
- profile + dropdown

**Comment by jquispe-oddball** [comment]
[PR submitted](https://github.com/department-of-veterans-affairs/vets-website/pull/33429)
@candilemoine what do you mean by profile and dropdown? The dropdown is working on the header

**Comment by seipeljl** [comment]
@jquispe-oddball @candilemoine we received some feedback from VA stakeholders during our design review yesterday that having POA Request as the stand-alone nav item for Pilot felt confusing (it was unclear whether this was navigation vs. some sort of breadcrumb or page header).  We agree, and would like to move Get Help down as the 2nd item in the nav, directly to the right of POA Requests.  We'll update designs.  Please let me know if you have any questions or concerns.  So sorry for the last-minute change!

**Comment by jquispe-oddball** [comment]
@seipeljl I pushed the update up as part of my pr review, see below:

<img width="1086" alt="Image" src="https://github.com/user-attachments/assets/73e59d2f-4d45-4433-a015-0101e6ad9ba0" />

**Comment by seipeljl** [comment]
@jquispe-oddball looks great!  cc: @eheintzelman @Kate-allbee

**Comment by eheintzelman** [comment]
@seipeljl Should "requests" in "POA requests" be capitalized?

**Comment by seipeljl** [comment]
@eheintzelman @jquispe-oddball Yes, I believe it should read **POA Requests** since it's part of the main navigation, which according to the [content style guide](https://design.va.gov/content-style-guide/capitalization#exceptions-where-we-use-title-capitalization) suggests we'd want to use title case.

**Comment by Kate-allbee** [comment]
Only note was what was mentioned by Eva in [this slack thread](https://dsva.slack.com/archives/C06ABHUNBRS/p1736890576773689)

**Comment by Kate-allbee** [comment]
Just noticed that in the designs, we did not include an active state for the "POA Requests" in the navigation, which should look like this with the blue line underneath:

<img width="302" alt="Image" src="https://github.com/user-attachments/assets/4ece5db6-e255-439d-8230-c58530020c59" />

I can create a new ticket for this to be added.

**Comment by eheintzelman** [comment]
@Kate-allbee A few other notes
- we also need a hover state for the navigation items (default, hover, active)
- Also take a look at recommended states for the dropdown items. What does hover and active state look like if Profile is selected? 
- I'm seeing bolding for the main navigation items (POA Requests, etc.) This is not in the designs. It is in the design for the secondary nav. 
- left-alignment issue as you flagged
- the sizing of the profile icon and dropdown icon look off (a bit small)
- I would like to see some spacing between the user name and the profile icon on the left; and then the profile name and the dropdown icon on the right. 
- Nav items should be in title case. "POA Requests" instead of "POA requests"
- Is the design following [guidance on spacing units](https://design.va.gov/foundation/spacing-units) using multiples of 8? I'm seeing margin and padding that don't follow this and wondering if this might be making alignment more of a challenge. 

In general, can we point to the va.gov header as the model to follow. There may be just a small number of exceptions to flag (like active and hover state). Styling will change with the visual design that Michael is doing but that should just be color selections.

**Comment by eheintzelman** [comment]
@Kate-allbee also check mobile responsiveness.

**Comment by Kate-allbee** [comment]
@jquispe-oddball Is the main nav extra bolded somehow? I see it is supposed to be bold in what's showing in the design system, but on staging it's looking a little extra bold: 

![Image](https://github.com/user-attachments/assets/f0d8ba24-be31-42c3-843f-63c3648f9d52)

**Comment by jquispe-oddball** [comment]
@Kate-allbee as discussed, closing this ticket since there is a new ticket with all the new changes, content updates from midpoint review
https://github.com/department-of-veterans-affairs/va.gov-team/issues/101461
